server/proxy/list.go

```diff
@@ -56,7 +57,7 @@ func (p *Proxy) GetInstalledApp(appID apps.AppID, checkEnabled bool) (*apps.App,
 }
 
 func (p *Proxy) PingInstalledApps(ctx context.Context) (installed []apps.App, reachable map[apps.AppID]bool) {
-	all := p.store.App.AsMap()
+	all := p.store.App.ListIncludeDisabled()
```

Comment:
Why do we want to ping them if they're disabled?

Reply:
I think it's used in `apps list` to show their online status; 0/5 kinda makes sense.